api/types/container/secret.go

```diff
@@ -0,0 +1,12 @@
+package container
+
+import "os"
+
+type ContainerSecret struct {
```

Comment:
Shouldn't containers have a secret reference, then access the data through a secrets.SecretGetter?